netctl/commands.go

```diff
@@ -168,27 +174,37 @@ var Commands = []cli.Command{
 				Name:      "ls",
 				Aliases:   []string{"list"},
 				Usage:     "List rules for a given tenant/policy",
-				ArgsUsage: "[policy]",
-				Flags:     []cli.Flag{tenantFlag, allFlag, jsonFlag},
-				Action:    listRules,
+				ArgsUsage: "",
+				Flags: []cli.Flag{tenantFlag, allFlag, jsonFlag, quietFlag,
+					cli.StringFlag{
+						Name:  "policy, p",
+						Usage: "Policy - REQUIRED",
+					},
+				},
+				Action: listRules,
 			},
 			{
 				Name:      "rm",
 				Aliases:   []string{"delete"},
 				Usage:     "Delete an existing rule.",
-				ArgsUsage: "[policy] [rule id]",
-				Flags:     []cli.Flag{tenantFlag},
-				Action:    deleteRule,
+				ArgsUsage: "[rule id]",
```

Comment:
@vijayvikrant @jainvipin 
Bringing up the broader topic of policy and rule UX here. Just because policy and rules are different REST objects, user doesn't have to see it that way. We can present the rules as a sub-object of policy. If we had a UI, it would display a table of rules inside the policy object(In fact this is what our rudimentary web-ui does).

Given that, I think the right way to present this to the user is:
netctl policy create <policy-name>
netctl policy add-rule <policy-name> <rule-id> [<rule-options>]
netctl policy delete-rule <policy-name> <rule-id> [<rule-options>]
netctl policy list-rule <policy-name> OR netctl policy inspect <policy-name>
netctl policy rm <policy-name>

I based add-rule and delete-rule options here similar to "docker network connect" and "docker network disconnect" CLI. This will be more easier for people to visualize rules and policies.